app/models/policy_configuration.rb

```diff
@@ -12,6 +12,26 @@
 class PolicyConfiguration < ApplicationRecord
   ACADEMIC_YEAR_REGEXP = /\A20\d{2}\/20\d{2}\z/.freeze
 
+  SERVICES = [
```

Comment:
Arguably these properties should probably be moved to columns on the database records as a next step. We already have `policy_types` being equivalent to the `policies` key here.

Reply:
PolicyConfiguration only has a list of `policy_types` otherwise currently there is no other way of associating it to what routing_name they belong to. But we can't store the routing_name in the db because then we can't boot the routes.rb file without querying the DB, which is the noob error I hit earlier on CI.